sites/next.skeleton.dev/tailwind.config.js

```diff
@@ -1,27 +1,26 @@
-// import { skeleton } from '../../packages/skeleton/dist/plugin/index.cjs'
```

Comment:
We'll need to keep these comments or it may break HRM

Reply:
Added it back.